packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md

```diff
@@ -0,0 +1,672 @@
+---
+title: SCIM 1.1 Protocol Reference
+icon: /assets/img/icons/scim.svg
+meta:
+  - name: description
+    content: Your SCIM API must support specific SCIM 1.1 API endpoints to work with Okta. Those endpoints and their explanations are detailed here.
+---
+
+# Okta and SCIM 1.1 Protocol
+
+This reference focuses on how Okta API endpoints share information with SCIM Version 1.1 specific API calls.
```

Comment:
Is there somewhere a reader can go to get an overview of what this is and how it works? If so, we should link to it here.